src/internal/components/pluggableVisualizations/pivotTable/PluggablePivotTable.tsx

```diff
@@ -599,6 +437,37 @@ export class PluggablePivotTable extends AbstractPluggableVisualization {
         return generateDimensions(mdObject, VisualizationTypes.TABLE);
     }
 
+    private adaptPropertiesToMdObject(
+        visualizationProperties: IVisualizationPropertiesWrapper,
+        mdObject: VisualizationObject.IVisualizationObjectContent,
+    ) {
+        // This is sanitization of properties from KD vs current mdObject from AD
+        const columnWidths = getColumnWidthsFromProperties(visualizationProperties);
+        if (columnWidths) {
+            this.sanitizeColumnWidths(columnWidths, mdObject);
+        }
+        // TODO INE: if sorting is saved in KD Widget add sanitization of sorting
```

Comment:
what to do with this TODO?

Reply:
Removed TODO comment. As we agreed with @ivanmjartan and @ivan-nejezchleb - we will not touch sorting from visualizationWidget